python/hail/keytable.py

```diff
@@ -74,6 +74,11 @@ def __repr__(self):
         return self._jkt.toString()
 
     @staticmethod
+    @handle_py4j
+    @typecheck(hc=HailContext,
+               rows_py=listof(anytype),
```

Comment:
Leaving out a param from typecheck raises a `RuntimeError`, which is what we want (and why your test failed I think)